cmd/podman/search.go

```diff
@@ -177,39 +172,47 @@ func getSystemContextAndRegistries(c *cli.Context) ([]string, *types.SystemConte
 			return nil, nil, errors.Wrapf(err, "error getting registries to search")
 		}
 	}
-
-	// If user flagged to skip verify for HTTP connections, set System Context as such
+	skipTLS := make([]bool, len(registries))
```

Comment:
Why not make a bool map based on the registies and then set it true for all registries, Then below you will not need to do any search

Reply:
map[string]bool

Reply:
oh shoot yeah that's a good idea